backend/events/models.py

```diff
@@ -6,8 +6,9 @@
 class Event(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField(blank=True, null="True")
-    start_time = models.DateTimeField(null="False")
-    end_time = models.DateTimeField(null="False")
+    start_date = models.DateField(blank=True, null="True")
+    start_time = models.TextField(null="True", blank=True)
```

Comment:
Also, I'm pretty sure `null=True`, rather than `null="True"`, is necessary for that to work properly.